### ISSUE TITLE
Parentheses get eaten by search

### ISSUE THREAD

**Issue by m-linssen** [KELLIA/dictionary#108]
Try a search for ⲉⲓⲛⲉ (ⲛ-, ⲙⲙⲟ⸗) and you'll see that the parentheses get removed before executing the search, resulting in no result (pun intended)

https://coptic-dictionary.org/entry.cgi?tla=C928 is still there, of course

This is new behaviour, since today even if memory serves me correctly, or at the very least this was still working a few days ago

**Comment by amir-zeldes** [comment]
This has been the case more or less since the dictionary went online, it hasn't been changed. The expectation is that it's much more likely users will try to find something using Coptic letters when the entry contains parentheses than the other way around, so the collated search index totally ignores them.

**Comment by m-linssen** [comment]
Ah, reproduction. The mother of any solution to any issue! Reproduce, reproduce, reproduce: always and always reproduce before reacting to what a user says. And usually, a good bit of time is spent on dragging the exact circumstances and facts out of him / her.

You and I are both right, and wrong. I am right about parentheses not being an impediment to search (although I formulated that rather unluckily), and you are right about them always having been removed.
I never noticed that they were, as the result of a search was in line with my expectations.
And you didn't notice that such wasn't the root cause to the specific problem perceived by me in this issue, because you didn't reproduce

There is a bug here, albeit minor I suppose - I haven't extrapolated.
A search for ⲉⲓⲛⲉ (ⲛ-, ⲙⲙⲟ⸗) will lead to nothing.
A search for ⲃⲱⲕ (ⲉⲃⲟⲗ ϩⲛ-) will lead to the expected result

ⲃⲱⲧⲥ (ⲉⲃⲟⲗ ⲉ-,   ⲟⲩⲃⲉ-, ⲛⲉⲙ) fails, ⲉⲓⲛⲉ (ⲉϫⲛ- (ⲧⲟⲟⲧ⸗,   ϭⲓϫ)) fails: the comma sure is posing issues here

Good luck fixing it!

**Comment by amir-zeldes** [comment]
Hi again, yes, the comma is causing those searches to fail, but I'm not sure why that would be a problem. A user would normally search for ⲉⲓⲛⲉ; I don't think I've ever tried typing exactly:

`ⲉⲓⲛⲉ (ⲛ-, ⲙⲙⲟ⸗)`

into the search box. It's not very likely as a use case because it would require the user to already know exactly how the entry is spelled in the dictionary... Trying to support searching for comma would also be complicated somewhat by comma being an ambiguous code point for Coptic/English, and the way the quick search box works is by splitting the input on whitespace, diagnosing what parts are Coptic, then searching for the words, searching in definitions, or a combination of both.

If you're trying to link to specific entries, I would suggest using the TLA ID: https://coptic-dictionary.org/entry.cgi?tla=C928

**Comment by phoenix-mossimo** [comment]
This problem has been resolved with #30: 

- forms like ⲉⲓⲛⲉ (ⲛ-, ⲙⲙⲟ⸗), ⲃⲱⲧⲥ (ⲉⲃⲟⲗ ⲉ-, ⲟⲩⲃⲉ-, ⲛⲉⲙ) have been split into separate forms: 
```
          <form type="compound" xml:id="CF3200"><orth>ⲉⲓⲛⲉ (ⲛ-)</orth></form>
          <form type="compound" xml:id="CF_TMP221"><orth>ⲉⲓⲛⲉ (ⲙⲙⲟ⸗)</orth></form>
```
```
          <form xml:id="CF2163">
            <usg type="geo">B</usg>
            <orth>ⲃⲱⲧⲥ (ⲉⲃⲟⲗ ⲉ-)</orth>
          </form>
          <form>
            <usg type="geo">B</usg>
            <orth>ⲃⲱⲧⲥ (ⲉⲃⲟⲗ ⲟⲩⲃⲉ-)</orth>
          </form>
          <form>
            <usg type="geo">B</usg>
            <orth>ⲃⲱⲧⲥ (ⲉⲃⲟⲗ ⲛⲉⲙ-)</orth>
          </form>
```

These are now displayed as such on the test-server and are findable with or without parenthesis search: 
https://corpling.uis.georgetown.edu/coptic-dictionary-dev/entry.cgi?tla=C928 
https://corpling.uis.georgetown.edu/coptic-dictionary-dev/entry.cgi?tla=C566
